hooks/setup_window.py

```diff
@@ -137,29 +134,20 @@ def createWindow():
                                 context's shotgun instance
                 """
                 app = self.parent
-                project = app.context.project
-                sg = app.context.tank.shotgun
-                # set filters and search fields for entity type "Project"
-                filters=[["id", "is", project['id']],]
-                fields=["sg_width", "sg_height"]
-                result=sg.find_one("Project", filters, fields)
-                # with result, set parameters accordingly or use default otherwise
-                if result:
-                    videoWidth = result.get("sg_width", DEFAULT_WIDTH)
-                    videoHeight = result.get("sg_height", DEFAULT_HEIGHT)
+                
+                videoWidth = cmds.getAttr("defaultResolution.width")
+                videoHeight = cmds.getAttr("defaultResolution.height")
 
-                # Find first camera matching pattern and set as active camera
-                # if not use default current active camera
-                camera_name_pattern = app.get_setting( "camera_name_pattern", "persp" )
-                cameraList = [c.name() for c in pm.ls(type="camera", r=True) if re.search( camera_name_pattern, c.name() )]
-                if not "cam" in MODEL_EDITOR_PARAMS.keys() and cameraList:
-                    MODEL_EDITOR_PARAMS["cam"] = cameraList[0]
+                cameraTrans = cmds.modelEditor( 'modelPanel4', q=True, cam=True )
```

Comment:
Not a big fan of hard coded `modelPanel4`

You could go for something like:
```suggestion
                panelName = cmds.getPanel(withFocus=True)
                if panelName not in cmds.getPanel(type="modelPanel"):
                    message = "Please select a viewport before trying to render"
                    self.logger.error(message)
                    QtGui.QMessageBox.Critical(None, "No Viewport selected", message)
                    raise RuntimeError(message)

                cameraTrans = cmds.modelEditor(panelName, q=True, cam=True)
```

Otherwise you can just implement the `camera_name_pattern` settings in `info.yml`  which they had before